app/assets/stylesheets/cieszyn.scss

```diff
@@ -9,9 +13,23 @@ body.cieszyn {
     text-align: justify;
   }
 
-  table {
+  .warranty {
     @extend .table;
     @extend .table-hover;
     @extend .table-striped;
   }
+
+  .prices {
+    @extend .table;
+  }
+
+  .prices.checked {
+    background-color: $selected-row-color;
+  }
+
+  .prices th {
```

Comment:
Merge rule `.prices th` with rule on line 22<br>Selector should have depth of applicability no greater than 2, but was 3